src/shadowbox/server/main.ts

```diff
@@ -81,15 +81,16 @@ function main() {
 
   logging.info('Starting...');
   const userConfigFilename = getPersistentFilename('shadowbox_config.json');
-  createManagedAccessKeyRepository(
+  createServerAccessKeyRepository(
       proxyHostname, new FilesystemTextFile(userConfigFilename), shadowsocksServer, stats)
       .then((managedAccessKeyRepository) => {
-        const managerService = new ShadowsocksManagerService(
-            serverConfig, managedAccessKeyRepository, stats);
+        const managerService =
+            new ShadowsocksManagerService(serverConfig, managedAccessKeyRepository, stats);
```

Comment:
I think you want to rename `managedAccessKeyRepository` -> `serverAccessKeyRepository`?

Reply:
Changed to `accessKeyRepository`